service/src/main/java/uk/gov/hmcts/reform/fpl/service/FurtherEvidenceNotificationService.java

```diff
@@ -54,13 +54,7 @@ public Set<String> getDesignatedLocalAuthorityRecipients(CaseData caseData) {
     }
 
     public Set<String> getDesignatedLocalAuthorityRecipientsOnly(CaseData caseData) {
-        final RecipientsRequest recipientsRequest = RecipientsRequest.builder()
-            .caseData(caseData)
-            .secondaryLocalAuthorityExcluded(true)
-            .legalRepresentativesExcluded(true)
-            .build();
-
-        return localAuthorityRecipients.getRecipients(recipientsRequest);
+        return new HashSet<>(localAuthorityRecipients.getDesignatedLocalAuthorityContacts(caseData));
```

Comment:
is this going to change any behaviour elsewhere in the application?

Reply:
`getDesignatedLocalAuthorityRecipientsOnly` is a new method added by CFV release, which I was just copying `getDesignatedLocalAuthorityRecipients` right above and modify it a bit.

Unless we are using some proxy bean provided by some dependency / api, I guess LocalAuthorityRecipientsService is the actual implementation we are using? It just doing the same thing, but returning a fallback inbox if no recipient address is found